astropy/io/votable/tree.py

```diff
@@ -6,16 +6,19 @@
 import codecs
 import gzip
 import io
+import json
 import os
 import re
 import urllib.request
+from pathlib import Path
 
 # THIRD-PARTY
 import numpy as np
 from numpy import ma
 
 # LOCAL
 from astropy import __version__ as astropy_version
+from astropy.coordinates import FK4, FK5, ICRS, AltAz, Galactic, Supergalactic
```

Comment:
You might have to localize this import. I think this somehow triggers some circular import.

Reply:
You mean doing

```python
from ...coordinates.builtin_frames import FK4, FK5, ICRS, AltAz, Galactic, Supergalactic
``` 

?

Reply:
It fails locally

Reply:
No, I mean import it within method that needs it instead of top of module.

Reply:
It works :open_mouth: